docs/api-docs/analytics/BODL.md

```diff
@@ -0,0 +1,432 @@
+
+  sampleAnalyticsProvider.load('<%= property_id %>');
+  sampleAnalyticsProvider.page();
+
+  // Advanced Matching
+  if (BODL.customer && BODL.customer.id) {
+    var customerObj = {
+      email: BODL.customer.email,
+    }
+
+    if (BODL.customer.phone) {
+      var phoneNumber = BODL.customer.phone;
+      if (BODL.customer.phone.indexOf('+') === -1) {
+        // No country code, so default to US code
+        phoneNumber = `+1${phoneNumber}`;  
+      }
+
+      customerObj.phone = phoneNumber;
+    }
+
+    sampleAnalyticsProvider.identify(BODL.customer.id, customerObj);
+  }
+}(window, document, 'sampleAnalyticsProvider');
+</script>
+```
+
+### Add to cart
+
+The following snippet covers tracking the Add to Cart event, actively tracking each new product ID, and quantity added.
+
+```javascript
+<!-- Sample Pixel Code Start: Product Detail Page Add to Cart Event -->
+<script>
+document.querySelectorAll('[data-cart-item-add]').forEach(form => form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const formData = new FormData(event.target);
+  let productId, productQty;
+  for (const pair of formData.entries()) {
+    if (pair[0] === 'product_id') {
+      productId = pair[1];
+    } else if (pair[0] === 'qty[]') {
+      productQty = pair[1];
+    }
+  }
+
+  // References:
+  // https://developer.bigcommerce.com/theme-objects/product
+  // https://developer.bigcommerce.com/stencil-docs/developing-further/catalog-price-object
+  sampleAnalyticsProvider.instance('<%= property_id %>').track('AddToCart', {
+    content_id: BODL.product.id,
+    content_category: BODL.breadcrumbs[1] ? BODL.breadcrumbs[1].name : '',
+    content_type: 'product_group',
+    content_name: BODL.product.title,
+    quantity: productQty,
+    price: BODL.product.price.without_tax.value,
+    value: (BODL.product.price.without_tax.value * productQty),
+    currency: BODL.product.price.without_tax.currency,
+  });
+}}));
+
+if (BODL.cartItemAdded) {
+  sampleAnalyticsProvider.instance('<%= property_id %>').track('AddToCart', {
+    content_id: BODL.cartItemAdded.product_id,
+    content_type: 'product_group',
+    content_name: BODL.cartItemAdded.name,
+    quantity: BODL.cartItemAdded.quantity,
+    price: BODL.cartItemAdded.price.value,
+    value: BODL.cartItemAdded.total.value,
+    currency: BODL.cartItemAdded.price.currency,
+  });
+</script>
+<!-- Sample Pixel Code End: Product Detail Page Add to Cart Event -->
+  ```
+
+### Add to wishlist
+The following snippet is for tracking when users add an item to their wishlist. Note that this sends individual products specified by the 'added_product_id' and a commented section regarding category data. We do not track category data on the wishlist, but we included it for future-proofing.
```

Comment:
```suggestion
The following snippet is used to track when users add an item to their wishlist. Note that this sends individual products specified by the 'added_product_id' and a commented section regarding category data. We do not track category data on the wishlist, but the snippet includes it for future-proofing.
```

Reply:
I made all the changes except in the first sentence. I did not want to change the text to "is used" because it is passive voice.